openapi/components/schemas/Cashier/CashierRequest.yaml

```
@@ -17,8 +17,8 @@ properties:
     $ref: ../CustomerId.yaml
   strategyId:
     description: |-
-      ID of a strategy to be applied for this request for empty `amounts` and `customAmount`.
-      If it is not specified, randomly selected strategy with a matching `filter` value is applied.
+      ID of a strategy to be applied for this request for `amounts` and `customAmount`.
+      If it is not specified, randomly selected strategy with a matching `filter` value is applied, but only for empty `amounts` and `customAmount`.
```

suggestion: I think it reads better if we remove the last comma
```suggestion
      If this field is not specified, a randomly selected strategy with a matching `filter` value is applied for empty `amounts` and `customAmount`.
```